.gitignore

```diff
@@ -47,3 +46,14 @@ bower.json
 *.json
 *.csv
 
+#ignore analysis files
```

Comment:
that was left over from a previous .gitignore within the repo. I can change that if you want.